### PR TITLE
Start to use GResource features

### PR DESCRIPTION
**Pull request by ricotz**

### CODE REVIEW

File: lib/Utils.vala [hunk 1]
```diff
@@ -332,5 +333,53 @@ namespace Gala
...
+					warning (e.message);
+					return null;
+				}
+			}
+
+			return resize_pixbuf;
+		}
+
+		/**
+		 * Creates a new reactive ClutterActor at 36px with the close pixbuf
```
Comment (donadigo) [inline lib/Utils.vala]: This should be ...at 36px with the **resize** pixbuf

Comment (ricotz) [inline lib/Utils.vala]: Isn't it?

File: lib/Utils.vala [hunk 2]
```diff
@@ -332,5 +333,53 @@ namespace Gala
...
+		 *
+		 * @return The resize button actor
+		 */
+		public static Clutter.Actor create_resize_button ()
+		{
+			var texture = new Clutter.Texture ();
```
Comment (donadigo) [inline lib/Utils.vala]: I do not like the use of `Clutter.Texture` here. It is deprecated and we should use `Clutter.Image` instead.

Comment (ricotz) [inline lib/Utils.vala]: Replacing deprecated symbols is not part this. While there is already a wide usage of Clutter.Texture there is no harm in following the current style.

**Pull request merged**